lib/plugins/json_body_parser.js

```diff
@@ -25,7 +25,7 @@ function jsonBodyParser(options) {
     var override = options.overrideParams;
 
     function parseJson(req, res, next) {
-        if (req.getContentType() !== 'application/json' || !req.body) {
```

Comment:
I considered doing that first, but I didn't want to modify the output in case there was reliance on it. On second thought, I can't imagine anyone depending on that output being case-sensitive.